projects/core/src/store-finder/store/reducers/index.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ActionReducerMap,
+  createFeatureSelector,
+  MemoizedSelector,
+  MetaReducer
+} from '@ngrx/store';
+
+import * as fromFindStores from './find-stores.reducer';
+import * as fromViewAllStores from './view-all-stores.reducer';
+
+import { InjectionToken, Provider } from '@angular/core';
+import { StoresState } from '../store-finder-state';
+
+export function getStoreFinderReducers(): ActionReducerMap<StoresState> {
```

Comment:
I think we can remove 'StoreFinder' from all function names. They are inside 'StoreFinderModule', so no need add 'StoreFinder' in names. 
Please also check all other changes.